scopesim/tests/tests_server/test_database.py

```diff
@@ -18,6 +18,12 @@ def mock_client():
         yield client
 
 
+@pytest.fixture(scope="class")
+def all_pkg():
+    # TODO: This could use some kind of mock to avoid server access
+    yield db.get_all_package_versions()
+
+
```

Comment:
Ah, I think I wanted to do somethings like that at some point, but then forgot 👍